### PR TITLE
hw19, jdbc, Insert/Update/Delete

### PR DESCRIPTION
**Pull request by u2107680007911543774e7r**
ready

### CODE REVIEW

File: src/test/java/lesson_jdbc_01/CustomerDaoImplIntegrationTest.java [hunk 1]
```diff
@@ -1,48 +1,76 @@
...
+		assertTrue(customerDao.insertCustomer(INSERT_DELETE_CUSTOMER));
+		assertEquals(expected, customerDao.findCustomersByCustNum(INSERT_DELETE_CUSTOMER.getCustNum()));
+	}
+
+	@Test
+	public void test2UpdateCustomer() throws SQLException {
+		Customer expected = UPDATE_CUSTOMER;
+		assertTrue(customerDao.updateCustomer(UPDATE_CUSTOMER));
+		assertEquals(expected, customerDao.findCustomersByCustNum(UPDATE_CUSTOMER.getCustNum()));
+
```
Comment (un-install) [inline src/test/java/lesson_jdbc_01/CustomerDaoImplIntegrationTest.java:68]: Tests is good

File: src/main/java/lesson_jdbc_01/CustomerDaoImpl.java [hunk 2]
```diff
@@ -46,4 +46,80 @@
...
+
+		int rowsChanged = stmt.executeUpdate();
+		if (rowsChanged > 0) {
+			stmt.close();
+			conn.close();
+			return true;
+		}
+		stmt.close();
+		conn.close();
+		return false;
```
Comment (un-install) [inline src/main/java/lesson_jdbc_01/CustomerDaoImpl.java]: Too many rows

**Review by un-install — APPROVED** [review]
Good

File: src/main/java/lesson_jdbc_01/CustomerDaoImpl.java [hunk 3]
```diff
@@ -46,4 +46,80 @@
...
+		Connection conn = ConnectToDB.getConnection();
+		String sql = "DELETE FROM customers WHERE cust_num = ?";
+		PreparedStatement stmt = conn.prepareStatement(sql);
+		stmt.setBigDecimal(1, id);
+
+		int rowsChanged = stmt.executeUpdate();
+		if (rowsChanged > 0) {
+			stmt.close();
+			conn.close();
+			return true;
```
Comment (un-install) [inline src/main/java/lesson_jdbc_01/CustomerDaoImpl.java]: too many rows

File: src/test/java/lesson_jdbc_01/CustomerDaoImplIntegrationTest.java (hunk 1 above)
Comment (SerhiiPasko) [inline src/test/java/lesson_jdbc_01/CustomerDaoImplIntegrationTest.java:68]: In this case you must also test how findCustomersByCustNum work. You have tests that depends to other method

**Review by SerhiiPasko — APPROVED** [review]
Good. Read comment

**Pull request merged**